Revised CTA Banner CSS changes - 579

nagaraju-gspann opened the pull request:
Fix #579

Test URLs:
- Before: https://main--danaher-abcam--aemsites.hlx.page/en-us/page-test/cta-description
- After: https://579-CTA-CSS-Changes--danaher-abcam--aemsites.hlx.live/en-us/page-test/cta-description

shivangi1422 commented on eds/blocks/cta-banner/cta-banner.js:
```diff
@@ -6,25 +6,24 @@ import { applyClasses } from '../../scripts/scripts.js';
-  block.classList.add(...'cta-container flex md:items-center flex-col md:flex-row justify-between gap-8 md:gap-8 w-full p-6 mt-10 mb-14 rounded-xl bg-[#edf6f7]'.split(' '));
+  block.classList.add(...'cta-container flex md:items-center flex-col md:flex-row justify-between gap-6 md:gap-8 w-full p-6 mt-10 mb-14 rounded-xl bg-[#edf6f7]'.split(' '));
   const title = block.querySelector('h3');
-  applyClasses(title, 'title mt-0 mb-4 !leading-[22px] !font-bold text-xl text-[#378189] break-words');
```
make text-xl as important i.e !text-xl. It is taking the styling for h3 rather than text-xl

nagaraju-gspann commented:
- Modified CSS changes according to revised Figma.
- Addressed & Fixed the review comments.
- Pushed and Raised PR.  

CC: @Meenu-GSPANN , @shivangi1422, @pardeepgera23

pardeepgera23 commented on eds/blocks/cta-banner/cta-banner.js:
```diff
@@ -1,30 +1,29 @@
   const productFullResponse = await getProductsListResponse(1);
   const parsedJson = JSON.parse(productFullResponse || '{}');
-  block.classList.add(...'cta-container flex md:items-center flex-col md:flex-row justify-between gap-8 md:gap-8 w-full p-6 mt-10 mb-14 rounded-xl bg-[#edf6f7]'.split(' '));
+  block.classList.add(...'cta-container flex md:items-center flex-col md:flex-row justify-between gap-6 md:gap-8 w-full p-6 mt-10 mb-14 rounded-xl bg-[#edf6f7]'.split(' '));
```
Use applyClasses from scripts.js

nagaraju-gspann replied:
Fixed review comments.
